mlflow/sklearn/utils.py

```diff
@@ -26,6 +28,9 @@
 
 _SAMPLE_WEIGHT = "sample_weight"
 
+# A organized template to store sklearn metrics
```

Comment:
Nit: I think we could be more specific with this comment. _SklearnMetric represents a metric that will be computed and logged during the autologging routine for a particular model type